src/ixdat/techniques/ms.py

```diff
@@ -326,6 +326,31 @@ def integrate_signal(self, mass, tspan, tspan_bg, ax=None):
             ax.fill_between(t, S_bg, S, color=STANDARD_COLORS[mass], alpha=0.2)
         return np.trapz(S - S_bg, t)
 
+    def integrate_flux(self, mol, tspan, tspan_bg, ax=None):
+        """Integrate a calibrated ms signal with background subtraction and evt. plotting
+
+        TODO: Should this, like grab_signal does now, have the option of using a
+            background saved in the object rather than calculating a new one?
```

Comment:
I don't know. What do you think? Would you find it more useful or more confusing?

Reply:
Yes I think that would be convenient. You set the background initially and then never have to think about it again. 

Note, this should already be part of main. I think I submitted a separate PR a while back, because this was useful in a wider context than just for deconvolution